javascript/new-src/src/constants.js

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
No, it's because the header gets stripped for app.js so we don't end up with a load of license headers.